### PR TITLE
Add landing page #1

### PR DESCRIPTION
**Pull request by Lomino5**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,6 +6,6 @@
     <title>Document</title>
 </head>
 <body>
-    
+    <h1>This is our landing page</h1>
```
Comment (JanLehner) [inline index.html:9]: Could you make it more welcoming?

**Comment by Lomino5** [comment]
ok

**Review by JanLehner — CHANGES_REQUESTED** [review]
Please make it more welcoming

**Review by JanLehner — APPROVED** [review]
Very nice

**Pull request merged**